src/query/api/v1/middleware/request_classification.go

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2021 Uber Technologies, Inc.
```

Comment:
I renamed query_inspection.go -> request_classification.go, but GH is treating it as a delete and new file which makes the diff pretty crappy :(